From 3cafaf98caa0bfe2324b94a0d3da7b0b1c78145e Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Thu, 17 Nov 2016 00:55:48 +0000
Subject: [PATCH 5/5] themes/slider: Vertically center the pie_countdown

Prior to this change we could end up with a warped pie countdown if the
buttons were of a larger size, breaking the constraints of the pie to be
vertically stretched.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 src/themes/slider/theme.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/themes/slider/theme.c b/src/themes/slider/theme.c
index 46c32cd..d7d4823 100644
--- a/src/themes/slider/theme.c
+++ b/src/themes/slider/theme.c
@@ -722,6 +722,7 @@ void add_notification_action(GtkWindow* nw, const char* text, const char* key, A
 		if (!windata->pie_countdown) {
 			windata->pie_countdown = gtk_drawing_area_new();
 			gtk_widget_set_halign (windata->pie_countdown, GTK_ALIGN_END);
+			gtk_widget_set_valign (windata->pie_countdown, GTK_ALIGN_CENTER);
 			gtk_widget_show(windata->pie_countdown);
 
 			gtk_box_pack_end (GTK_BOX (windata->actions_box), windata->pie_countdown, FALSE, TRUE, 0);
-- 
2.10.2

